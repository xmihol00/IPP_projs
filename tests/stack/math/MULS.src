.IPPcode21

DEFVAR GF@num1
DEFVAR GF@num2

PUSHS int@10
PUSHS int@3
MULS
POPS GF@num1
WRITE GF@num1
WRITE string@\010

PUSHS float@0x1.6p+2
PUSHS float@-0x1.9p+2
MULS
POPS GF@num2
WRITE GF@num2
WRITE string@\010


PUSHS GF@num2
PUSHS GF@num1
INT2FLOATS
MULS
POPS GF@num2
WRITE GF@num2
WRITE string@\010

PUSHS GF@num2
FLOAT2INTS
PUSHS int@-3
MULS
POPS GF@num1
WRITE GF@num1
WRITE string@\010
