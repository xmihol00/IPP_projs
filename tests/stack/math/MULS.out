30
-0x1.1300000000000p+5
-0x1.01d0000000000p+10
3093
